packages/polymath-ui/src/components/EthNetworkWrapper/actions.js

```diff
@@ -170,6 +171,11 @@ export const init = (networks: Array<string>) => async (dispatch: Function) => {
     web3WS,
   };
 
+  await polyClient.connect({
```

Comment:
Aren't the modern sagas handling this?

Reply:
Not for this scenario. The sagas do this based on Modern's routing which doesn't exist for when replacing components.